ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+
+# Copyright 2014 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Alba Health Check Module
+"""
+
+import subprocess
+import hashlib
+import time
+import uuid
+import json
+import os
+
+from ovs.extensions.healthcheck.utils.extension import Utils
+from ovs.dal.lists.albabackendlist import AlbaBackendList
+from ovs.log.healthcheck_logHandler import HCLogHandler
+from ovs.dal.lists.albanodelist import AlbaNodeList
+from ovs.dal.lists.servicelist import ServiceList
+from ovs.extensions.generic.system import System
+
+
+class AlbaHealthCheck:
+    """
+    A healthcheck for Alba storage layer
+    """
+
+    def __init__(self, logging=HCLogHandler(False)):
+        """
+        Init method for Alba health check module
+
+        @param logging: ovs.log.healthcheck_logHandler
+
+        @type logging: Class
+        """
+
+        self.module = "alba"
+        self.utility = Utils()
+        self.LOGGER = logging
+        self.show_disks_in_monitoring = False
+        self.machine_details = System.get_my_storagerouter()
+        self.machine_id = System.get_my_machine_id()
+        self.temp_file_loc = "/tmp/ovs-hc.xml"  # to be put in alba file
+        self.temp_file_fetched_loc = "/tmp/ovs-hc-fetched.xml"  # fetched (from alba) file location
+        self.temp_file_size = 1048576  # bytes
+
+    @staticmethod
+    def _fetch_available_backends():
+        """
+        Fetches the available alba backends
+
+        @return: information about each alba backend
+
+        @rtype: list that consists of dicts
+        """
+
+        result = []
+        for abl in AlbaBackendList.get_albabackends():
+
+            # check if backend would be available for vpool
+            for preset in abl.presets:
+                available = False
+                if preset.get('is_available'):
```

Comment:
if 'is_available' would have any value besides a False'ish value, this will always be True
Solution:
```if preset.get('is_available') is True```